autoapi/templates/python/property.rst

```diff
@@ -1,4 +1,12 @@
 {%- if obj.display %}
+
```

Comment:
The issue here comes when documenting a class. If the `property.rst` or the `method.rst` use "=" as header, the following structure is achieved:

```rst
Foo
===
...

bar_property
============

.. py:property:: ...
```

This generates two independent sections instead of a nested structure.

Properties and methods should have 1 more level depth for their header if they are rendered inside a class.

Thus, I decided not to include any headers for these objects.

Reply:
I didn't explain myself very clearly. The importance is where the whitespace is rather than what the sections are.
In my first code snippet, there isn't a blank line before the section header, and there is only one line between the section header and the directive.
In the second code snippet, there is no blank line before the directive.
Is that how things are output? If not, please could we change the templates to reflect that.

Reply:
Thanks for indicating this, Ashley. I got your point now. Let me ensure that the templates introduce the less whitespace amount as possible.